### PR TITLE
Remove last use of UIWebView

### PR DESCRIPTION
**Pull request by vanwagonet**
no for realz this time.

refs: none
affects: none
release note: none

### CODE REVIEW

File: rn/Teacher/package.json [hunk 1]
```diff
@@ -54,7 +54,6 @@
     "react-native-button": "^2.1.0",
     "react-native-camera": "^3.23.1",
     "react-native-counter": "Kerumen/react-native-counter",
-    "react-native-device-info": "^0.10.1",
```
Comment (cobbal) [inline rn/Teacher/package.json:57]: We weren't even ever using the webview?

Comment (vanwagonet) [reply rn/Teacher/package.json:57]: This package used UIWebView to get the user agent for the device. At one point we must have been using this package, but apparently we haven't been for a while.

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/15yzpf1) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F15yzpf1)
teacher: [Link](https://rebrand.ly/szq4rnw) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fszq4rnw)
parent: [Link](https://rebrand.ly/6g8e8sx) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F6g8e8sx)

**Comment by vanwagonet** [comment]
We used to use the react-native-device-info package to get the version number for display on the RN profile drawer. When we made that native we removed the last use of the package.

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: None

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.65% | 93.65% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**